README.md

```diff
@@ -14,9 +14,336 @@ ManageIQ plugin for the Hawkular provider.
+```
+
+And then we add the upstream remotes to fetch changes on the main codebase:
+
+```bash
+cd ~/ManageIQ
+
+git -C ./manageiq remote add upstream git@github.com:ManageIQ/manageiq
+git -C ./manageiq-ui-classic remote add upstream git@github.com:ManageIQ/manageiq-ui-classic
+git -C ./manageiq-schema remote add upstream git@github.com:ManageIQ/manageiq-schema
+git -C ./manageiq-api remote add upstream git@github.com:ManageIQ/manageiq-api
+git -C ./manageiq-providers-hawkular remote add upstream git@github.com:ManageIQ/manageiq-providers-hawkular
+git -C ./hawkular-client-ruby remote add upstream git@github.com:hawkular/hawkular-client-ruby
+```
+
+Now we need to make ManageIQ use our vendored repositories instead of the local
+clone it does with Bundler. Create a file on `bundler.d` on the core repository
+called `vendor.rb` (any name is OK, but we're going for consistency here).
+
+This file is our addition to the Gemfile where we can override some gems and
+add some other development dependencies without messing with the main
+repository. Its contents are the following:
+
+```ruby
+override_gem name, path: File.expand_path("../../manageiq", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-ui-classic", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-schema", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-api", __dir__)
+override_gem name, path: File.expand_path("../../manageiq-providers-hawkular", __dir__)
+```
+
+We also recommend some other dependencies that might come in hand for debugging:
+
+```ruby
+gem 'pry'
+gem 'pry-byebug'
+```
+
+The next step is to configure each repository:
+
+```bash
+cd ~/ManageIQ/manageiq
+./bin/setup
+
+cd ../manageiq-ui-classic
+./bin/setup
+
+cd ../manageiq-schema
+./bin/setup
+
+cd ../manageiq-api
+./bin/setup
+
+cd ../manageiq-providers-hawkular
+./bin/setup
+
+cd ../hawkular-client-ruby
+bundle install
+```
+
+This should be enough. To run the application, we can run this, which will run the application on background on port 3000:
+
+```bash
+cd ~/ManageIQ/manageiq
+bundle exec rake evm:start
+```
+
+Then, we can visit `http://localhost:3000` and sign in. If everything is working fine, you should see this:
+
+![](screenshots/login.png)
+
+Sign in with the default credentials (`admin`/`smartvm`), and you should be good to go!
+
+### Updating the repositories
+
+To update the repositories, you need to go on each one and run `bin/update`. This script takes care of it:
+
+```bash
+#!/bin/bash
+
+MIQ_DIR="$HOME/ManageIQ"
+OLD_PWD="$(pwd)"
+
+cd $MIQ_DIR
+
+for repo in $(find $MIQ_DIR -maxdepth 1 -type d)
+do
+  echo "Updating repo $repo..."
+
+  cd $repo
+
+  if [ -d .git ]; then
+    git fetch -a
+    git pull --rebase upstream master
+    ./bin/update
+    cd -
+  fi
+done
+
+echo "Updating core..."
+
+cd $MIQ_DIR/manageiq
+
+./bin/update
+
+cd $OLD_PWD
+
+echo "DONE."
+```
+
+## Installing Docker
```

Comment:
I'm not very, very sure because I don't remember well... But IIRC, docker-compose is installed separately from the engine. That should be mentioned here because hawkinit depends on compose.

Reply:
That's true, will add it.